examples/1-basic/galaxy-redshift-distributions.ipynb

```diff
@@ -0,0 +1,645 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "height": 432,
+       "width": 567
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.plot(z, nz)\n",
+    "plt.xlabel(\"$z$\")\n",
+    "plt.ylabel(\"$n(z)$\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Simulation setup\n",
+    "\n",
+    "For the simulation (GLASS or otherwise), we have a given set of redshift shells. Here, we use a comoving distance grid as an arbitrary example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "zgrid = glass.distance_grid(cosmo, 0.0, 3.0, num=20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We then have to set the radial profile of the shells. Here, we use a tophat profile in redshift as our first example. Later, we will see how this impacts the results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "shells = glass.tophat_windows(zgrid)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These are all the ingredients that define the radial discretisation of the simulation. For a GLASS simulation, these are usually user choices; for *N*-body simulations, they might instead be given."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Matching the redshift distribution\n",
+    "\n",
+    "With the (matter or lensing) shells defined, the clustering or lensing of each sampled galaxy is fully determined by its associated shell. The only choice is *how many* galaxies are assigned to each shell: having relatively more galaxies in a specific shell will make the clustering or lensing signal from that shell become more prominent in the final galaxy sample.\n",
+    "\n",
+    "**The redshifts assigned to each individual galaxies are completely irrelevant here. The clustering and lensing signals are fully determined by the shells.**\n",
+    "\n",
+    "Still, we expect that if we assign galaxies to shells according to their desired $n(z)$, we will recover the clustering and lensing signals to the best of our abilities.\n",
+    "\n",
+    "Fortunately, we can predict exactly how this happens, at least for linear galaxy bias."
```

Comment:
I am a bit confusing with the role that `glass.partition` has. The N(z) we produce in the previous cells was normalised to 1, right? So is the result from `partition` really the number of galaxies. Later on, `nz_eff` is introduced, as a result from `glass.combine`, which is shown to match the first N(z) we produced in the first place. 

This is all done before any matter shells are simulated. So I am a bit confused about what we are trying to show in this section. I am sure it is important, but I don't think I can say I understand it.

Reply:
I tried to address this in the text by always talking about the **relative** number of galaxies in each shell.